lib/openstudio-standards/standards/Standards.PlantLoop.rb

```diff
@@ -55,7 +55,7 @@ def chw_sizing_control(model, chilled_water_loop, dsgn_sup_wtr_temp, dsgn_sup_wt
   # @param model [OpenStudio::Model::Model] OpenStudio model object
   # @return [String] common_pipe or heat_exchanger
   def plant_loop_set_chw_pri_sec_configuration(model)
-    pri_sec_config = 'common_pipe'
+    pri_sec_config = 'heat_exchanger'
```

Comment:
Yes, that is correct. This would have wider consequences. I thought this `plant_loop_set_chw_pri_sec_configuration` was a bit goofy but didn't want to break convention. Would we prefer to limit this to a subset? I'm not sure on the intended scope of this update.